src/Native/Runtime/EHHelpers.cpp

```diff
@@ -301,7 +303,7 @@ EXTERN_C Int32 RhpPInvokeExceptionGuard()
 }
 #endif
 
-#if defined(_AMD64_) || defined(_ARM_) || defined(_X86_) || defined(_ARM64_)
+#if defined(_AMD64_) || defined(_ARM_) || defined(_X86_) || defined(_ARM64_) || defined(_WASM_)
```

Comment:
If this ifdef serving any purpose since you do not seem to be able to use it for not-yet-implemented WASM EH? If it is not useful, delete it instead.

Reply:
This one is a little bit useful since it provides an empty implementation for RhpThrow* for new architectures. Since we're using the portable runtime, the bootstrapper is our implementation for wasm at the moment.